integreat_cms/cms/templates/pages/page_form.html

```diff
@@ -167,288 +167,16 @@ <h1 class="heading">
                     {% include "../hix_widget.html" %}
                 {% endif %}
                 {% if page %}
-                    <div class="mt-4 lg:mt-0 rounded border border-blue-500 shadow-2xl bg-white">
-                        <div class="rounded p-4 bg-water-500">
-                            <h3 class="heading font-bold text-black">
-                                <i icon-name="feather" class="pb-1"></i>
-                                {% translate "Minor edit" %}
-                            </h3>
-                        </div>
-                        <div class="w-full px-4 pb-4 rounded shadow bg-white">
-                            <label>{% minor_edit_label request.region language %}</label>
-                            {% render_field page_translation_form.minor_edit %}
-                            <label for="{{ page_translation_form.minor_edit.id_for_label }}"
-                                   class="secondary">
-                                {{ page_translation_form.minor_edit.label }}
-                            </label>
-                            <div class="help-text">{% minor_edit_help_text request.region language page_translation_form %}</div>
-                        </div>
-                    </div>
+                    {% include "./page_form_sidebar/_page_minor_edit_template.html" with box_id="page-minor-edit" %}
```

Comment:
```suggestion
                    {% include "./page_form_sidebar/minor_edit_box.html" with box_id="page-minor-edit" %}
```
I think "template" is kind of redundant - since this file is already in the `templates` directory and ends with `.html`, I guess it's clear that it's a template (the same goes for the other boxes).